apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -42,12 +38,55 @@ TODO: Fix asset loading in environments that don't use `pathPrefix`. This is a t
 ## Without branding
 
 ```jsx live
-<VideoPlayer branding={false} title="GitHub media player">
+<VideoPlayer hasBranding={false} title="GitHub media player">
+  <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" />
+  <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" />
+</VideoPlayer>
+```
```

Comment:
FYI, I'm still seeing the branding here

![Uploading Screenshot 2024-07-17 at 15.03.21.png…]()

Reply:
Oops, good spot, thank you!

Fixed in 054a5dbd7793479e803139843bbd5a8f947026d0